.devcontainer/Dockerfile

```diff
@@ -36,49 +37,31 @@ RUN apt-get -fy install git vim emacs sudo \
     docker.io \
     iputils-ping net-tools \
     libncurses5 \
-    libpython2.7
-
-RUN groupadd -g $USER_GID $USERNAME
-RUN useradd -s /bin/bash -u $USER_UID -g $USER_GID -G docker -m $USERNAME
-RUN echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME
-RUN chmod 0440 /etc/sudoers.d/$USERNAME
-
-RUN mkdir -p /var/downloads
-RUN cd /var/downloads
-RUN curl -JL https://github.com/microsoft/vscode-cpptools/releases/download/0.27.0/cpptools-linux.vsix > extension.zip
-RUN unzip extension.zip
-RUN mkdir -p /home/$USERNAME/.vscode-server/extensions
-RUN mv extension /home/$USERNAME/.vscode-server/extensions/ms-vscode.cpptools-0.27.0
-RUN mkdir -p /home/$USERNAME/bin
-RUN curl https://raw.githubusercontent.com/restyled-io/restyler/master/bin/restyle-path -o /home/$USERNAME/bin/restyle-path
-RUN chmod +x /home/$USERNAME/bin/restyle-path
-RUN chown -R $USERNAME:$USERNAME /home/$USERNAME
-RUN echo "PATH=/home/$USERNAME/bin:${PATH}" >> /home/$USERNAME/.bashrc
-
-# $USERNAME needs to own the esp-idf and tools for the examples to build
-RUN chown -R $USERNAME:$USERNAME /opt/espressif/esp-idf
-RUN chown -R $USERNAME:$USERNAME /opt/espressif/tools
-
-# $USERNAME needs to own west configuration to build nRF Connect examples
-RUN chown -R $USERNAME:$USERNAME /opt/NordicSemiconductor/nrfconnect/
-
-# allow read/write access to header and libraries
-RUN chown -R $USERNAME:$USERNAME /opt/ubuntu-21.04-aarch64-sysroot/usr/
-
-# allow licenses to be accepted
-RUN chown -R $USERNAME:$USERNAME /opt/android/sdk
-
-# AmebaD requires access to change build_info.h
-RUN chown -R $USERNAME:$USERNAME /opt/ameba/ambd_sdk_with_chip_non_NDA/
-
-# NXP uses a patch_sdk script to change SDK files
-RUN mkdir -p /opt/sdk/sdks/
-RUN chown -R $USERNAME:$USERNAME /opt/sdk/sdks/
-
-RUN chown -R $USERNAME:$USERNAME /opt/fsl-imx-xwayland/5.15-kirkstone/
-
-# Add access to openocd for VSCode debugging
-RUN chown -R $USERNAME:$USERNAME /opt/openocd
+    libpython2.7 \
```

Comment:
Should we add `apt-get clean && rm -rf /var/lib/apt/lists/` to release the package cache?

Reply:
If this is in vscode devcontainer I am unsure ... since this is local, we probably do not need to minimize the size of the image anymore, so this would only be useful to decrease disk usage on the host. 

Is this worth it (slight delay ... image building is already slow) but more disk saved on the assumption we do not install/update packages in the image?

Reply:
ok, makes sense